packages/api/src/controllers/asset.ts

```diff
@@ -931,6 +951,8 @@ app.patch(
       storage = await reconcileAssetStorage(req, asset, storage);
     }
 
+    playbackPolicy = validateAssetPlaybackPolicy(playbackPolicy);
```

Comment:
Actually asset playback policies **cannot** be changed. We need to disallow changing the type from `public` to `lit_signing_condition` (and vice versa) because that would require moving the asset files between object stores. We also cannot allow changing the `resourceId` of the asset as that would give control of the account owner over the creator resource. 

The right logic here is to disallow the changes mentioned above, and only allow updating the `unifiedAccessControlConditions` field instead.